libs/lz4/src/main/java/org/elasticsearch/lz4/ESLZ4Decompressor.java

```diff
@@ -20,11 +20,18 @@
 
 import net.jpountz.lz4.LZ4Exception;
 import net.jpountz.lz4.LZ4FastDecompressor;
+import net.jpountz.util.ByteBufferUtils;
+import net.jpountz.util.SafeUtils;
```

Comment:
I think that we should be linking to our version here? We fork this so that we can use a custom `org.elasticsearch.lz4.SafeUtils`. It may not actually matter since I think it is mostly using this class here for basic methods `checkRange`, `readByte`, etc and not the actual modified ones `readInt`, `writeShortLE`, `readShortLE`. But just in case I think we should be using our modified version?

Reply:
pushed https://github.com/elastic/elasticsearch/pull/139221/commits/08c3662f7c43483371f7198c706f658925092cda